public/kibana-services.ts

```diff
@@ -40,9 +40,9 @@ export const [getVisualizationsPlugin, setVisualizationsPlugin] = createGetterSe
 export const [getNavigationPlugin, setNavigationPlugin] = createGetterSetter<
   NavigationPublicPluginStart
 >('NavigationPlugin');
-export const [getErrorOrchestrator, setErrorOrchestrator] = createGetterSetter<ErrorOrchestrator>(
-  'ErrorOrchestrator'
-);
+export const [getErrorOrchestrator, setErrorOrchestrator] = createGetterSetter<
+  ErrorOrchestratorService
+>('ErrorOrchestratorService');
```

Comment:
I don't undestand what is the reason to use a `createGetterSetter` for this.

Reply:
The idea is that we can use the orchestrator throughout the app without having to create a new instance every time, like the other services (toast, https, plugins, etc)

Reply:
I think that is a good approach to get the common services from one place, you need to only know from where import `kibana-services` (maybe it need a new name)

Reply:
Also, maybe not for this case where the instance is static but for another service that needs a constructor for its instance is good to get the same instance from a getterSetter